content/desktop/hardened-desktop/_index.md

```diff
@@ -27,18 +27,33 @@ grid:
 >
 >Hardened Docker Desktop is available to Docker Business customers only.
 
+## What is Hardened Docker Desktop?
+
 Hardened Docker Desktop is a group of security features for Docker Desktop, designed to improve the security of developer environments without impacting developer experience or productivity.
 
-It is for security conscious organizations who don’t give their users root or admin access on their machines, and who would like Docker Desktop to be within their organization’s centralized control.
+It lets admins define and enforce robust security settings. It guarantees that developers and the containers they deploy are unable to intentionally or unintentionally circumvent these settings. Additionally, Hardened Docker Desktop enhances container isolation, which helps mitigate potential security threats, and ensures that malicious payloads cannot breach the Docker Desktop Linux VM and the underlying host.
+
+## Who is it for?
+
+It is for security conscious organizations who:
+- Don’t give their users root or admin access on their machines
+- Would like Docker Desktop to be within their organization’s centralized control 
+- Have certain compliance obligations
 
-Hardened Docker Desktop moves the ownership boundary for Docker Desktop configuration to the organization, meaning that any security controls admins set cannot be altered by the user of Docker Desktop.
+## What does Hardened Docker Desktop include?
 
-Hardened Docker Desktop includes:
+It includes:
 - Settings Management, which helps admins to confidently manage and control the usage of Docker Desktop within their organization.
-- Enhanced Container Isolation, a setting that instantly enhances security by preventing containers from running as root in Docker Desktop’s Linux VM and ensures that any configurations set using Settings Management cannot be bypassed or modified by containers.
-- Registry Access Management, which allows admins to control the registries developers can access.
-- Image Access Management, which gives admins control over which images developers can pull from Docker Hub.
+- Enhanced Container Isolation (ECI), a setting that instantly enhances security by preventing containers from running as root in Docker Desktop’s Linux VM and ensures that any configurations set using Settings Management cannot be bypassed or modified by containers.
+- Registry Access Management (RAM), which allows admins to control the registries developers can access.
+- Image Access Management (IAM), which gives admins control over which images developers can pull from Docker Hub.
+
+### How does it help my organisation?
+
+Hardened Desktop features work independently but collectively to create a defense-in-depth strategy, safeguarding developer workstations against attacks across various functional layers, such as configuring Docker Desktop, pulling container images, and running container images. This multi-layered defense approach ensures comprehensive security.
```

Comment:
> safeguarding developer workstations against attacks across various functional layers, such as configuring Docker Desktop, pulling container images, and running container images.

Those things are not attacks per se but can pose threats or go against internal policies, not sure how to word it though 

> running container images

Let's change to "running containers from malicious images" or similar

Reply:
have rephrased to say 'potential' attacks. Second suggestion doesn't work as its talking about the various entry points for potential attacks